spreadGun.js

```diff
@@ -0,0 +1,30 @@
+class SpreadGun extends Weapon
```

Comment:
Why not make all guns in a guns.js file?

Reply:
One file, one class, I don't like to put multiple classes into one file. The file structure should be reworked but I would not do this in the scope of this story